src/main/java/org/takes/rs/RsWithStatus.java

```diff
@@ -94,6 +94,9 @@ public InputStream body() throws IOException {
 
     /**
      * Make head.
+     * @todo #160:DEV To implement concatenation and selection with the
+     *  conjunction Concat and Select class to get rid of List.add()
+     *  between line 118 to line 127
```

Comment:
I'm not sure I understand what exactly are you talking about here. Please, explain more what do you want me, as a future fixer of this puzzle, to do